Makefile

```diff
@@ -35,11 +35,12 @@ clear:
 	rm -rf _out
 
 .PHONY: install
-install: static-release
+install:
 	sudo install -D -m 755 bin/containerd-nydus-grpc /usr/local/bin/containerd-nydus-grpc
 	sudo install -D -m 755 misc/snapshotter/nydusd-config.${FS_DRIVER}.json /etc/nydus/config.json
 	sudo install -D -m 644 misc/snapshotter/nydus-snapshotter.${FS_DRIVER}.service /etc/systemd/system/nydus-snapshotter.service
-	sudo systemctl enable /etc/systemd/system/nydus-snapshotter.service
+
+	@if which systemctl; then sudo systemctl enable /etc/systemd/system/nydus-snapshotter.service;fi
```

Comment:
If no systemd is installed, any error should be thrown?

Reply:
no need

Reply:
The integration test needs this Makefile target and the base container image has no systemd packed. The user should know if he/she has systemd installed as a system administrator.